patches/tradeSequence.py

```diff
@@ -267,14 +267,15 @@ def patchMermaid(rom):
 
 def patchMermaidStatue(rom):
     rom.patch(0x18, 0x095D, 0x0962, "", fill_nop=True)
-    rom.patch(0x18, 0x0966, 0x097A, ASM("""
+    rom.patch(0x18, 0x0966, 0x0971, ASM("""
```

Comment:
0971 would end in the middle of an `ld a, XX` instruction (end addresses for these types of patches are excluding that address)

Reply:
ok, merging the patch with the adjacent patch should resolve this concern, but looking at the generated rom, the NOP from the earlier patch started at 0971. will merge the patches for cleanliness.